flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/factories/HiveCatalogFactoryTest.java

```diff
@@ -57,10 +59,10 @@
 	public ExpectedException expectedException = ExpectedException.none();
 
 	@Test
-	public void test() {
+	public void test() throws IOException {
```

Comment:
Create a new test case for this

Reply:
`test` is too generic, please give a meaningful name for this test